website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,128 @@
+It also works through Kubernetes to request needed assets from the cloud provider.
+
+Here are some attributes of the Karpenter provisioner:
+
+* **Unschedulable pods**: Karpenter only looks at pods that the Kubernetes scheduler identifies as unschedulable.
+If the Kubernetes scheduler is able to schedule a pod, the Karpenter provisioner takes no action on it.
+* **Provisioner CRD**: Provisioner CRDs define how the provisioner controller behaves.
+Each CRD contains constraints that impact the nodes that can be provisioned and attributes of those nodes (such timers for removing nodes).
+See [Provisioner CRD](/docs/provisioner-crd/) for a description of settings.
+* **Well-known labels**: The CRD can use well-known Kubernetes labels to allow pods to request only certain instance types, computer architectures, operating systems, or other attributes when creating nodes.
+* **Time to live**: A provisioner can also include time-to-live values to indicate when nodes should be decommissioned after a set amount of time from when it was created or after it becomes empty of deployed pods.
+* **Multiple provisioner CRDs**: An important feature of Karpenter is that it allows multiple provisioner CRDs on the same cluster.
+One possible result is that you could configure different teams on the same cluster to run on completely separate capacity.
+One team could run on GPU nodes while another uses storage nodes, with each having a different set of defaults.
+This approach could some day lead Karpenter to allow provisioners with different cloud providers on the same cluster or a control plane on one cloud and data plane on another.
+
+### Karpenter scheduler
+
+Besides creating nodes, Karpenter also works with the Kubernetes schedular to schedule pods.
+To avoid conflicts, Karpenter only schedules pods that the Kubernetes scheduler has marked unschedulable.
+When Karpenter finds unschedulable pods, it provisions the needed nodes and immediately binds the previously unschedulable pods to the new nodes.
+Once Karpenter brings up a node, that node is available for the Kubernetes scheduler to schedule pods on it as well.
+
+### Cloud provider
+Karpenter makes requests to provision new nodes to the associated cloud provider.
+The first supported cloud provider is AWS, although Karpenter is designed to work other cloud providers as well.
+Separating Kubernetes and AWS-specific settings allows Karpenter a clean path to integrating with other cloud providers.
+
+While using Kubernetes well-known labels, the provisioner can set values that are specific to the cloud provider.
+So, for example, to include a certain instance type, you could use the Kubernetes label `node.kubernetes.io/instance-type`, but set its value to an AWS instance type (such as `m5.large` or `m5.2xlarge`).
+Other cloud assets you can identify include security groups and capacity type.
```

Comment:
You can't specify security groups via well known labels.

"Cloud providers may choose to implement support for additional vendor specific labels like `node.k8s.aws/capacity-type`"